packages/core-data/src/selectors.ts

```diff
@@ -686,6 +718,9 @@ export const __experimentalGetDirtyEntityRecords = createSelector(
 								entityConfig?.getTitle?.( entityRecord ) || '',
 							name,
 							kind,
+							hasMetaChanges: metadataKeys.includes( primaryKey )
```

Comment:
Looking at the existing logic, it feels like all the changes required in this selector could get moved to this line. I'm thinking about something like the following:

```js
hasMetaChanges: !! getEntityRecordNonTransientEdits(
	state,
	kind,
	name,
	primaryKey
).?meta,
```

All the conditions should be satisfied because these operations happen only for `primaryKeys` computed in the earlier stage.

Reply:
A related question is whether in the long run, the boolean property is going to be sufficient, or it would be preferred to list all individual modified meta changes. I was about to say that `hasMetaChanges` will become part of the public API, but I see that it's prefixed as experimental so it is less important. Anyway, it's good to answer the question if the boolean value can serve it's purpose in the far future.

Reply:
I was gonna share a similar question. I was wondering if we should abstract it to not include only `metaChanges` but track the different changes made to the post and do with that information whatever we consider.

I'm thinking that in the future we will probably have the possibility to edit "Post Data" like title, description, excerpt... And we might want to do something similar.

Reply:
I followed @gziolo's advice and simplified the logic 👍